### PR TITLE
microplane 0.0.21 (new formula)

### PR DESCRIPTION
**Pull request by nitrocode**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]
lgtm

File: Formula/microplane.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  sha256 "8deb72bdc72f96420decee7b680fa8f5feb48adc39f31e091e44a57734a4a575"
+  license "Apache-2.0"
+  head "https://github.com/Clever/microplane.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
```
Comment (gromgit) [inline Formula/microplane.rb:14]:
```suggestion
```
I'm pretty sure you want to compile for the same OS as the host, so setting `GOOS` should be unnecessary.

File: Formula/microplane.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dir = buildpath/"src/github.com/Clever/microplane"
+    dir.install buildpath.children
+    cd "src/github.com/Clever/microplane" do
+      system "make", "install_deps"
+      system "make", "build"
+      bin.install "bin/mp"
+      prefix.install_metafiles
```
Comment (dawidd6) [inline Formula/microplane.rb:21]:
```suggestion
```

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
squashed the commits before the merge (almost forgot, 😱 )

**Comment by nitrocode** [comment]
Ah I always forget to do that. Thanks for the check @chenrui333 !

**Comment by chenrui333** [comment]
> Ah I always forget to do that. Thanks for the check @chenrui333 !

no worries, thank you!!

**Comment by chenrui333** [comment]
@nitrocode, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by nitrocode** [comment]
Hmmm I don't quite understand this error message...

**Comment by chenrui333** [comment]
give it another shot

**Comment by chenrui333** [comment]
yeah 🎉

**Pull request closed without merging**